packages.json

```diff
@@ -2,6 +2,11 @@
   "packages": {
     "plugins": [
     {
+        "description" : "To Create Request File",
```

Comment:
You might consider writing a little more descriptive message here, from reading this and the github repo I have no idea what this plugin does 😸